main.qml

```diff
@@ -464,7 +464,7 @@ ApplicationWindow {
                         + (paymentId === "" ? "" : (qsTr("\nPayment ID: ") + paymentId))
                         + qsTr("\n\nAmount: ") + walletManager.displayAmount(transaction.amount)
                         + qsTr("\nFee: ") + walletManager.displayAmount(transaction.fee)
-                        + qsTr("\n\nMixin: ") + mixinCount
+                        + qsTr("\n\nRing size: ") + mixinCount
```

Comment:
@Jaqueeee I only know how to change words. If we need to change the actual calculation to ringsize=mixin+1 then I'll close this PR and you can do a better one?